server/sonar-server/src/test/java/org/sonar/server/computation/task/projectanalysis/step/SendIssueNotificationsStepTest.java

```diff
@@ -59,19 +61,26 @@
   static final String PROJECT_KEY = "PROJECT_KEY";
   static final String PROJECT_NAME = "PROJECT_NAME";
 
+  static final String BRANCH_COMPONENT_UUID = "BRANCH_UUID";
+  static final String BRANCH_COMPONENT_KEY = "BRANCH_KEY";
+  static final String BRANCH_COMPONENT_NAME = "BRANCH_NAME";
+  static final String BRANCH_NAME = "feature";
+
   static final long ANALYSE_DATE = 123L;
 
   static final Duration ISSUE_DURATION = Duration.create(100L);
   static final String ISSUE_ASSIGNEE = "John";
 
   static final Component PROJECT = builder(Type.PROJECT, 1).setUuid(PROJECT_UUID).setKey(PROJECT_KEY).setName(PROJECT_NAME).build();
+  static final Component BRANCH = builder(Type.PROJECT, 2).setUuid(BRANCH_COMPONENT_UUID).setKey(BRANCH_COMPONENT_KEY).setName(BRANCH_COMPONENT_NAME).build();
 
   @Rule
   public TreeRootHolderRule treeRootHolder = new TreeRootHolderRule()
     .setRoot(PROJECT);
 
   @Rule
   public AnalysisMetadataHolderRule analysisMetadataHolder = new AnalysisMetadataHolderRule()
+    .setBranch(null)
```

Comment:
null branch is only used on PR, it should be 
`setBranch(new DefaultBranchImpl)`